mcs/class/System.Private.CoreLib/System.Globalization/GlobalizationMode.Unix.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Globalization
+{
+	internal static partial class GlobalizationMode
```

Comment:
We try not to repeat the "main" type modifiers, so `partial class GlobalizationMode` should be fine

Reply:
Ok, will update that.